src/App.tsx

```diff
@@ -1,14 +1,131 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { ChangeEvent, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { ErrorType } from './types/ErrorType';
+import { ErrorNorification } from './Components/ErrorNorification';
+import { TodosList } from './Components/TodosList';
+import { getCurrentTime, getTodoId } from './utils/functionsHelper';
+import { SortBy } from './types/SortBy';
+import { TodoFooter } from './Components/TodoFooter';
 
-const USER_ID = 0;
+const USER_ID = 10632;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [typeOfError, setTypeOfError] = useState<ErrorType>(ErrorType.none);
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [filterBy, setFilterBY] = useState(SortBy.all);
+  const [countActiveTodos, setCountActiveTodos] = useState(0);
+
+  useEffect(() => {
+    const activeTodosCount = todos.reduce((count, cur) => {
+      if (!cur.completed) {
+        return count + 1;
+      }
+
+      return count;
+    }, 0);
+
+    setCountActiveTodos(activeTodosCount);
+  }, [todos]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await getTodos(USER_ID);
+
+        setTodos(data);
+      } catch (error) {
+        setTypeOfError(ErrorType.load);
+      }
+    };
+
+    fetchData();
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setNewTodoTitle(e.target.value);
+  };
+
+  const handleOnKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    if (e.key === 'Enter') {
+      if (!newTodoTitle.trim()) {
+        setTypeOfError(ErrorType.empty);
+      } else {
+        const newTodo = {
+          completed: false,
+          createdAt: getCurrentTime(),
```

Comment:
why do you need this?
createdAt and updatedAt this are database fields and are setted automatically